Point_set_3/include/CGAL/Point_set_3.h

```diff
@@ -465,8 +465,8 @@ class Point_set_3
     \note Properties of the added point other than its normal vector
     are initialized to their default value.
 
-    \note A normal property must have been added to the point set
-    before using this method.
+    \note If not already added, a normal property is automatically
+    added to the point set when using this method.
```

Comment:
There is
```
 \note Properties of the added point other than its normal vector
    are initialized to their default value.
```
just above. Does that remain true? It should probably clarify what is the value of normals when the normal map gets created with existing points?